### DISCUSSION TITLE
How to align the text of right click menu?

### DISCUSSION THREAD

**Discussion by i110077** [mbnuqw/sidebery#1559] [Q&A]
As shown in the picture, I want the text to be left-aligned to unify the UI style, and I tried to edit the CSS style many times but it doesn't work. Also is it possible to fix this issue in the new version, thanks for your efforts.
![sideberymenu](https://github.com/mbnuqw/sidebery/assets/83385874/a8823cd7-2471-4988-8ff0-0e4f2efcb954)

**Comment by mbnuqw** [comment]
- If you use native context menu (Sidebery settings > Context menu > Use native context menu), this can be [changed only with userChrome.css](https://www.reddit.com/r/FirefoxCSS/search?q=context+menu+align&restrict_sr=on&sort=relevance&t=all). Although, on my system (linux) titles are aligned. What OS are you using?
- If you use non-native context menu they should be aligned by default, so check if you have changed some custom style (Sidebery settings > Styles editor).

**Comment by i110077** [comment]
Thanks, the problem is solved, it was a problem on Windows, I set the Use native context menu. if you have the same problem, you can add the following code to userChrome.CSS: `#webextpanels-window popupset#mainPopupSet menupopup#contentAreaContextMenu menu label.menu-text {
    padding-left: 24px !important;
}`. Perfect. This is my favorite plugin, thanks again.
![01](https://github.com/mbnuqw/sidebery/assets/83385874/cb9cc374-8a07-4492-83b8-926184ea0c47)
